### DISCUSSION TITLE
blog/the-ultimate-terminal

### DISCUSSION THREAD

**Discussion by giscus** [angular-experts-io/angular-experts-website-comments#10] [General]
# blog/the-ultimate-terminal

A step by step guide to setup an epic terminal optimized for Angular development, with hyper, fish and oh-my-posh.

https://angularexperts.io/blog/the-ultimate-terminal

**Comment by mikeandtherest** [comment]
This is looking amazing, thanks for the article!
I wonder however how many chances will I have to make it work on a Windows environment.

**Reply by nivekcode** [reply]
Hi there, thx a lot. Very happy to hear that you liked the article. This definitely also works on windows. My colleague @tomastrajan uses windows and has it running. Unfortunately I am not a windows use and can not really assist you, maybe @tomastrajan can help?

**Reply by mikeandtherest** [reply]
Knowing that it works also under Win is great, I know at least that I wouldn't waste my time fighting with the windmills :) I'll give it a spin one of these days. Thanks!

**Reply by technbuzz** [reply]
Yeah it totally works with windows, Infact windows terminal looks cool just because we have ohmyposh :D. Scott hanselman has great article https://www.hanselman.com/blog/my-ultimate-powershell-prompt-with-oh-my-posh-and-the-windows-terminal. Even ohMyPosh has section for window installation.

**Comment by AlbePao** [comment]
HI Kevin, thanks for this guide, finally I've found a way to appreciate Hyper!

However I've noticed that Nerd Font doesn't support material font icons anymore, [as told in their cheat sheet](https://www.nerdfonts.com/cheat-sheet) and some of the icons in Angularic theme didn't work anymore, how can I fix them?

**Comment by AlbePao** [comment]
Hi @kreuzercode, the configuration file you have shown is not compatible with the latest versions of oh-my-posh. I've fix it and I would like to share the fix to update this article, also with the new icons for the angularic theme
